joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -351,9 +352,10 @@ controller_interface::return_type JointTrajectoryController::update(
         {
           if (!outside_goal_tolerance)
           {
-            auto res = std::make_shared<FollowJTrajAction::Result>();
-            res->set__error_code(FollowJTrajAction::Result::SUCCESSFUL);
-            active_goal->setSucceeded(res);
+            auto result = std::make_shared<FollowJTrajAction::Result>();
+            result->set__error_code(FollowJTrajAction::Result::SUCCESSFUL);
+            result->set__error_string("Goal reached, success!");
```

Comment:
```suggestion
            result->set__error_string("Goal successfully reached!");
```
Do we need to set the error string even in the case of success?

Reply:
I thought the same, but there is the **error** code `FollowJTrajAction::Result::SUCCESSFUL` so I thought we should populate the string as well ;)